### PR TITLE
[SDESK-956] - UI improvements in settings sections

### PR DESCRIPTION
**Pull request by akintolga**
[SDESK-957] - Match publish and subscribers
[SDESK-958] - Add titles
[SDESK-959] - Position of the "add new..." button
[SDESK-960] - Add a "show active only" filter for internal destinations

### CODE REVIEW

File: scripts/apps/internal-destinations/views/settings.html [hunk 1]
```diff
@@ -6,20 +6,27 @@ <h2 translate>Internal Destinations</h2>
             <button class="btn btn--primary btn--pull-right" ng-click="dest.create()">
                 <i class="icon-plus-sign icon-white"></i> {{ :: 'Add New Destination' | translate }}
             </button>
+
```
Comment (gbbr) [inline scripts/apps/internal-destinations/views/settings.html]: This blank line is not needed.

File: scripts/apps/content-filters/views/filter-search.html [hunk 2]
```diff
@@ -1,5 +1,10 @@
 <div ng-controller="FilterSearchCtrl">
-    <div class="panel-search">
+    <div class="split-content" >
```
Comment (gbbr) [inline scripts/apps/content-filters/views/filter-search.html]: Please improve the formatting in this file and make it more obvious which node nests what, using indentation.

File: scripts/apps/products/views/product-test.html [hunk 3]
```diff
@@ -1,4 +1,8 @@
-<div ng-controller="ProductsConfigCtrl">
+<div class="split-content products-content" ng-controller="ProductsConfigCtrl">
```
Comment (gbbr) [inline scripts/apps/products/views/product-test.html]: There are some indentation problems here too.

File: scripts/apps/publish/views/subscribers.html [hunk 4]
```diff
@@ -1,38 +1,42 @@
...
+        <div class="header">
+            <h2 translate>Subscribers</h2>
+            <div class="pull-right">
+                <div class="sortbar pull-left">
+                    <span translate>Filter:</span>
+                    <input type="text" class="input-medium" ng-model="search.name" ng-model-options="{debounce: 800}"/>
+                    <span translate>Target Type:</span>
+                    <select class="input-medium" ng-model="search.subscriber_type"
+                            ng-options="s.qcode as s.name for s in subTypes">
+                            <option value="" translate>Select</option>
```
Comment (gbbr) [inline scripts/apps/publish/views/subscribers.html]: I think this only needs to be indented 1 tab, like everything else.

**Comment by akintolga** [comment]
@gbbr Thanks for your review. I have updated the files to improve nesting. Let me know if you see any other issues.

**Pull request merged**